Added option to ask google for directions

andythorne opened the pull request:
### Short Description:

Added option to ask google for directions
### Changes:
- Added `google_directions` (boolean) config key. If set, the stepper will ask google directions API for the way (via walking) to the destination.
- Reduced long/lat randomness. each step could be altered by 9 feet (of factor 25), which sometimes sent you backwards. Now randoms to 3.64 feet (factor 10) by default.
- Closes #50 

Example in action:
![map](https://cloud.githubusercontent.com/assets/1301181/17233972/61da1e3c-552c-11e6-9c4f-4e4528a0f10a.png)

@OpenPoGo/maintainers

jmeyer2k commented:
Wow! Thanks for the commits! Make sure CI passes and then we'll get this merged.

epicblood commented:
could you also add this as a cli option?

andythorne commented:
Updated and passing!

jmeyer2k commented:
:+1:

wchill commented on README.md:
```diff
@@ -77,9 +78,10 @@ pip install -r requirements.txt
       -iv IV, --pokemon-potential IV                Set the ratio for the IV values to transfer (DEFAULT 0.4 eg. 0.4 will transfer a pokemon with IV 0.3)
       -ri, --recycle-items                          Recycle unneeded items automatically
-      -if LIST, --item-filter LIST                  Pass a list of unwanted items to recycle when collected at a Pokestop (e.g, [\"101\",\"102\",\"103\",\"104\"] to recycle potions when collected). Requires --recycle-items. 
+      -if LIST, --item-filter LIST                  Pass a list of unwanted items to recycle when collected at a Pokestop (e.g, [\"101\",\"102\",\"103\",\"104\"] to recycle potions when collected). Requires --recycle-items.
```
Should be removed

andythorne replied:
Done. That file needs to be cleaned up tho - lots of trailing whitespace all over it.

andythorne committed:
Added option to ask google for directions

Added google_directions (boolean) config key. If set, the stepper will
ask google directions API for the way (via walking) to the destination.

MaxLeiter commented:
Please modify the requirements.txt to include the googlemaps API

andythorne commented:
@MaxLeiter It's already there: https://github.com/andythorne/OpenPoGoBot/blob/dd001054aaee73fd72f6a8843b4edf5be6f1f0fd/requirements.txt#L8

MaxLeiter commented:
ah, thanks for pointing that out.

:+1: great work!

TCB13 commented:
What about a merge to dev?

MaxLeiter commented:
We don't have a dev branch, @TCB13. This is a different project

TCB13 commented:
@MaxLeiter sorry, github on iPhone, never again!
